### PR TITLE
Do not ignore every single file

### PR DESCRIPTION
**Pull request by paladox**

### CODE REVIEW

File: .eslintignore [hunk 1]
```diff
@@ -17,6 +17,5 @@ doc/
 build/
 demo/
 api/
-**/* *
```
Comment (nightwing) [inline .eslintignore]: this is needed to ignore files with space in the name

**Comment by nightwing** [comment]
Hi
Could you explain what is the goal of this change?
Thanks.

File: .eslintignore (hunk 1 above)
Comment (paladox) [inline .eslintignore]: Well this is ignoring all the files not specified as wanting to be linted.

Comment (nightwing) [inline .eslintignore]: How? if i add an error to makefile.dryice.js and run `node_modules/eslint/bin/eslint.js .` it finds that error.
This pattern should match only if the file has a space in name, unless there is a bug in eslint.

Comment (paladox) [inline .eslintignore]: Oh sorry.

**Pull request closed without merging**